src/Orchard.Web/Modules/Orchard.Projections/Migrations.cs

```diff
@@ -262,16 +266,26 @@ public class Migrations : DataMigrationImpl {
 
             ContentDefinitionManager.AlterTypeDefinition("ProjectionPage", cfg => cfg.Listable());
 
-            return 3;
+            return 4;
         }
 
         public int UpdateFrom2() {
-            SchemaBuilder.AlterTable("ProjectionPartRecord",
-                            table => table
-                                .AlterColumn("PagerSuffix", c => c.WithType(DbType.String).WithLength(255))
-                            );
+            SchemaBuilder.AlterTable("ProjectionPartRecord", table => table
+                .AlterColumn("PagerSuffix", c => c.WithType(DbType.String).WithLength(255))
+            );
 
             return 3;
         }
+
+        public int UpdateFrom3() {
+            SchemaBuilder.AlterTable("LayoutRecord", table => table
+                .AddColumn<string>("Guid", c => c.WithLength(64))

```

Comment:
I think we added support for the Guid type directly in SchemaBuilder. And a Guid is not 64, nit picking ;)